streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -440,8 +440,8 @@ public void shouldUseSpecifiedNameForTableSourceProcessor() {
                 topology,
                 expected,
                 expected + "-table-source",
-                "KSTREAM-SOURCE-0000000002",
-                "KTABLE-SOURCE-0000000003");
+                "KSTREAM-SOURCE-0000000004",
```

Comment:
These number changes are valid as 2 store names (internal, not visible for IQ) are generated.  I validated the topology description against a description generated by `2.2`

Reply:
I see. So the diff we're seeing is against tests that were previously modified (incorrectly) to match the non-incrementing behavior that you're fixing right now?

Reply:
Yes, that is correct.

Reply:
perfect. thanks for the reassurance!

Reply:
Actually, those test are newly added in a previous KIP-307 PR... But if Bill check agains 2.2, it should be fine.